x-pack/test/functional_solution_sidenav/tests/security_sidenav.ts

```diff
@@ -69,6 +69,14 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
 
         await expectNoPageReload();
       });
+
+      it('renders a feedback callout', async () => {
+        await solutionNavigation.sidenav.feedbackCallout.expectExists();
```

Comment:
Curious how does it render the callout in the tests? I'd expect tests not have a cloudTrialEndDate. Or do them?

Reply:
If you don't have a cloudTrialEndDate then you are not a trial user 😊 In that case the feedback button does show.

Reply:
makes sense! :)